pages/common/spatial.md

```diff
@@ -0,0 +1,27 @@
+# spatial
+
+> A set of commands for managing and developing SpatialOS projects.
+
+- Build workers for local deployment on Unity on Windows:
```

Comment:
"Build workers"? I'm not sure I understand here. Does this compile the source code for some kind of worker process?

Also, what does Unity have to do with an operating system?

Reply:
They build Unity (a game engine) processes that can run on a Windows' platform